### PR TITLE
Update Community for Mattermost manifest to 1.0.11

### PR DESCRIPTION
**Pull request by fmartingr**
## Summary
- Bump Community for Mattermost manifest version from 1.0.9 to 1.0.11
- Remove unused `defaultGroupCapability` from both manifests (app has only personal-scoped tabs, so this field had no effect)
- Fix static tab name to use `{{.AppName}}` template variable instead of hard-coded `"Mattermost"`
- Add `errorToString()` helper in `iframe.html.tmpl` to prevent `[object Object]` surfacing in auth error messages — applies to all `catch` blocks and `postMessage` payloads
- Add Jest tests for `errorToString` covering null/undefined, strings, `Error` instances, plain objects, and circular references

### CODE REVIEW

File: assets/iframe.html.tmpl [hunk 1]
```diff
@@ -32,6 +32,13 @@
       const domainRoot = '{{.SiteURL}}';
       const iframe = document.querySelector('iframe');
 
+      function errorToString(error) {
+        if (!error) return 'Unknown error';
+        if (typeof error === 'string') return error;
+        if (error.message) return error.message;
+        try { return JSON.stringify(error); } catch (_) { return String(error); }
+      }
```
[the remark this answers is not in the document]
Comment (wiggin77) [reply assets/iframe.html.tmpl:50]: fixed

File: webapp/src/errorToString.test.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Tests for the errorToString utility used in assets/iframe.html.tmpl.
+// The function is inlined in the template (no module system); keep both in sync.
+function errorToString(error) {
```
Comment (wiggin77) [inline webapp/src/errorToString.test.js:9]: If we must have a copy of it here, then there should be a check to ensure they do not get out of sync.

File: assets/iframe.html.tmpl [hunk 3]
```diff
@@ -103,13 +120,14 @@
             microsoftTeams.authentication.getAuthToken().then((token) => {
               resolve(token);
             }).catch((error) => {
-              console.error("Failed to get token:", error);
+              console.error("Failed to get token:", errorToString(error));
+              console.error(error); // Fallback to regular console.error in case the error coming from teams it's not a string
               if (retries < maxRetries) {
                 console.log("Retrying...");
                 retries++;
                 setTimeout(getToken, 250 + (250 * retries^2)); // Exponential backoff
```
Comment (wiggin77) [inline assets/iframe.html.tmpl]: Whoops.  This is using XOR instead of exponent, and results in no delay every time. 

```suggestion
                setTimeout(getToken, 250 + (250 * retries * retries)); // Exponential backoff
```

**Review by wiggin77 — APPROVED** [review]
LGTM 👍
